### PR TITLE
[systemtest] Add profiles for upgrade and olm-acceptance

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Enhancement

### Description

This small PR adds two new profiles to the `systemtest/pom.xml` file - `upgrade` and `olm_acceptance_upgrade`.
The profiles will be useful when running the tests on our automation. 

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

File: systemtest/pom.xml [hunk 1]
```diff
@@ -544,5 +552,18 @@
...
         </profile>
+
+        <profile>
+            <id>olm_acceptance</id>
+            <properties>
+                <skipTests>false</skipTests>
+                <groups>
+                    acceptance,
+                    olm,
+                    olmupgrade
```
[the remark this answers is not in the document]
Comment (im-konge) [reply systemtest/pom.xml:563]: renamed the profile to `olm_acceptance_upgrade`, so it makes more sense, thanks

**Review by see-quick — APPROVED** [review]
LGTM

**Comment by im-konge** [comment]
@strimzi-ci --help

**Comment by im-konge** [comment]
@strimzi-ci --help

**Comment by im-konge** [comment]
@strimzi-ci run tests --profile=upgrade

**Comment by im-konge** [comment]
@strimzi-ci run tests --profile=upgrade

**Review by Frawless — APPROVED** [review]

**Pull request merged**